game/resource/English/npc/tooltip_electrician.txt

```diff
@@ -74,7 +74,7 @@
 // talents
 "DOTA_Tooltip_Ability_special_bonus_electrician_energy_absorption_mana"         "+{s:value}% Energy Absorption Max Mana Absorbed"
 "DOTA_Tooltip_Ability_special_bonus_electrician_static_grip_damage"             "+{s:value_tooltip}% Static Grip Damage"
-"DOTA_Tooltip_Ability_special_bonus_electrician_electric_shield_damage"         "+{s:value} Electric Shield Damage"
+"DOTA_Tooltip_Ability_special_bonus_unique_electrician_electric_shield_damage"  "+{s:bonus_aura_damage} Electric Shield Damage"
```

Comment:
why are you changing the name?

Reply:
.... im not?

Reply:
oh the unique part, yeah that actually changes functionality

Reply:
you can thank valve

Reply:
https://github.com/OpenAngelArena/oaa/pull/3648#discussion_r1634007788

Reply:
stupid Valve....